christfellowshipchurch/web-app-v2
# Add discover filters

Adds discover filters to the discover page. 

TO TEST: 
Go to /discover and click around make sure everything feels pretty good. Core functionality should mostly mirror current https://christfellowship.church/discover.

![image](https://user-images.githubusercontent.com/2528817/111506551-ba07c100-8717-11eb-8ab0-cbe5d164cef6.png)
![image](https://user-images.githubusercontent.com/2528817/111506605-c9870a00-8717-11eb-8be8-b9cadff279e3.png)
